src/main/java/me/xmrvizzy/skyblocker/utils/Vec3Comparable.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2023 Skyblocker contributors
```

Comment:
This file is copied verbatim. Why do you put yourself as primary author.

Reply:
Not verbatim, there are some small changes to port to 1.20.x. If you want, I can put NEU first, but I thought convention is to put the most recent/current project first.

Reply:
Sorry, I have no ill intentions here. I didn't exactly "remove any reference to the original source", I even put NEU as the first line of copyright notices in each file now. I did change the rest to refer to Skyblocker, simply because it seemed weird to keep it as-is when it is now part of a different project, though you are right that the changes are (at least for now) mostly trivial indeed. I'm willing to add an additional notice to the copyright headers to clarify the situation further.

Reply:
Marking as resolved as the notice was adjusted and we discussed it on Discord.